.github/workflows/posix.yml

```diff
@@ -99,9 +99,21 @@ jobs:
       run: |
         mkdir -p local/openblas
         tar -C local/openblas --strip-components=2 -xf libs/openblas*.tar.gz
-        # do not package the static libs, they are ~55MB
-        rm local/openblas/*.a
-        python -m pip wheel -w dist -vv .
+        cp local/openblas/lib/libopenblas64_.so local/openblas/libopenblas_python.so
+        # do not package the static libs and symlinks, they are ~55MB
+        rm -rf local/openblas/lib/*
+        mv local/openblas/libopenblas_python.so local/openblas/lib/
+
+        cat <<EOF > run_in_docker.sh
+        cd /openblas
+        patchelf --set-soname libopenblas_python.so local/openblas/lib/libopenblas_python.so
```

Comment:
symlinks don't play well in wheels, they just end up as copies of the original files.
This hacks the soname & filename just to need the 1 file.
It would probably be better to handle this as an openblas build system patch than hacking this with `patchelf`.
I didn't take care of any ABI versioning in there.

Reply:
makes sense. Where did I use symlinks?

Reply:
> Where did I use symlinks?

You don't directly. But openblas build does the generic linux thing here, something along the lines of (I don't remember exactly the actual names):
libopenblas.so => symlink to libopenblas.so.0.3.3
libopenblas.so.3 => symlink to libopenblas.so.0.3.3
libopenblas.so.0.3.3 => the actual binary